app/src/main/java/tech/zafrani/companionforpubg/fragments/BaseFragment.java

```diff
@@ -2,7 +2,36 @@
 
 
 import android.app.Fragment;
+import android.os.Bundle;
+import android.support.annotation.LayoutRes;
+import android.support.annotation.Nullable;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import butterknife.ButterKnife;
+import butterknife.Unbinder;
 
 public abstract class BaseFragment extends Fragment {
 
+    private Unbinder unbinder = Unbinder.EMPTY;
+
+    @Nullable
```

Comment:
I don't think we should let this be nullable. If it is we should throw an `IllegalStateException` to catch it in development.